config/default.cfg

```diff
@@ -147,8 +147,10 @@ cfg$files2export$start <- c("config/conopt3.opt",
                             "scripts/input/climateAssessmentInterimRun.R",
                             "scripts/input/climateAssessmentImpulseResponse.R",
                             "scripts/input/run_KotzWenz_damages.R",
+                            "scripts/input/mag2rem.R",
                             ".Rprofile",
                             "config/mappingEDGEtoREMINDsectors.csv", # To do: Is this still needed?
+                            "config/mappingMAgPIE2REMIND.csv",
```

Comment:
why do we need this new mapping?

Reply:
It contains the mapping of the MAgPIE and REMIND variables and is used by the coupling script. It makes it easy to see what is transferred from MAgPIE and where it is written in REMIND. This makes it easy to update variable names if they change.